pls, fix chepach errors:
```

 Commit a45c2d7da4:
1: UC5 Commit title exceeds max length (80>75): "snippets: xen_dom0: qemu_cortex_a53: remove RAM and hypervisor node modification"

Commit 872deae700:
1: UC5 Commit title exceeds max length (76>75): "snippets: xen_dom0: spider_ca55: remove RAM and hypervisor node modification"

Commit 2ae233b61a:
1: UC5 Commit title exceeds max length (79>75): "snippets: xen_dom0: salvator_xs_m3: remove RAM and hypervisor node modification"
Error: Process completed with exit code 1.
```

--- a/snippets/xen_dom0/boards/rcar_h3ulcb_ca57.overlay
+++ b/snippets/xen_dom0/boards/rcar_h3ulcb_ca57.overlay
@@ -4,40 +4,4 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/delete-node/ &ram;
 /delete-node/ &scif2;
-
-/ {
-	/*
-	 * This node may differs on different setups, please check
-	 * following line in Xen boot log to set it right:
-	 * (XEN) Grant table range: 0x00000088080000-0x000000880c0000
-	 * Also, add extended region 2:
-	 * (XEN) [    0.928173] d0: extended region 2: 0x40000000->0x47e00000
-	 *
-	 * Xen passes actual values for setup in domain device tree, but Zephyr
-	 * is not capable to parse and handle it in runtime.
-	 */
-	hypervisor: hypervisor@88080000 {
-		compatible = "xen,xen";
-		reg = <0x0 0x88080000 0x0 0x40000 0x0 0x40000000 0x0 0x7e00000>;
-		interrupts = <GIC_PPI 0x0 IRQ_TYPE_EDGE IRQ_DEFAULT_PRIORITY>;
-		interrupt-parent = <&gic>;
-		status = "okay";
-	};
-
-	/*
-	 * This node may differs on different setups, because Xen picks
-	 * region for Domain-0 for every specific configuration. You can
-	 * start Xen for your platform and check following log:
-	 * (XEN) Allocating 1:1 mappings for dom0:
-	 * (XEN) BANK[0] 0x00000060000000-0x00000070000000 (256MB)
-	 *
-	 * Xen passes actual values for setup in domain device tree, but Zephyr
-	 * is not capable to parse and handle it in runtime.
-	 */
-	ram: memory@60000000 {
-		device_type = "mmio-sram";
-		reg = <0x00 0x60000000 0x00 DT_SIZE_M(256)>;
-	};
-};

--- a/snippets/xen_dom0/boards/rcar_salvator_xs_m3.overlay
+++ b/snippets/xen_dom0/boards/rcar_salvator_xs_m3.overlay
@@ -4,40 +4,4 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/delete-node/ &ram;
 /delete-node/ &scif2;
-
-/ {
-	/*
-	 * This node may differs on different setups, please check
-	 * following line in Xen boot log to set it right:
-	 * (XEN) Grant table range: 0x00000088080000-0x000000880c0000
-	 * Also, add extended region 2:
-	 * (XEN) [    0.928173] d0: extended region 2: 0x40000000->0x47e00000
-	 *
-	 * Xen passes actual values for setup in domain device tree, but Zephyr
-	 * is not capable to parse and handle it in runtime.
-	 */
-	hypervisor: hypervisor@88080000 {
-		compatible = "xen,xen";
-		reg = <0x0 0x88080000 0x0 0x40000 0x0 0x40000000 0x0 0x7e00000>;
-		interrupts = <GIC_PPI 0x0 IRQ_TYPE_EDGE IRQ_DEFAULT_PRIORITY>;
-		interrupt-parent = <&gic>;
-		status = "okay";
-	};
-
-	/*
-	 * This node may differs on different setups, because Xen picks
-	 * region for Domain-0 for every specific configuration. You can
-	 * start Xen for your platform and check following log:
-	 * (XEN) Allocating 1:1 mappings for dom0:
-	 * (XEN) BANK[0] 0x00000060000000-0x00000070000000 (256MB)
-	 *
-	 * Xen passes actual values for setup in domain device tree, but Zephyr
-	 * is not capable to parse and handle it in runtime.
-	 */
-	ram: memory@60000000 {
-		device_type = "mmio-sram";
-		reg = <0x00 0x60000000 0x00 DT_SIZE_M(256)>;
-	};
-};

--- a/snippets/xen_dom0/boards/rcar_spider_ca55.overlay
+++ b/snippets/xen_dom0/boards/rcar_spider_ca55.overlay
@@ -4,40 +4,4 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/delete-node/ &ram;
 /delete-node/ &hscif0;
-
-/ {
-	/*
-	 * This node may differs on different setups, please check
-	 * following line in Xen boot log to set it right:
-	 * (XEN) Grant table range: 0x00000078080000-0x000000780c0000
-	 * Also, add extended region 1:
-	 * (XEN) Extended region 1: 0x40000000->0x47e00000
-	 *
-	 * Xen passes actual values for setup in domain device tree, but Zephyr
-	 * is not capable to parse and handle it in runtime.
-	 */
-	hypervisor: hypervisor@78080000 {
-		compatible = "xen,xen";
-		reg = <0x0 0x78080000 0x0 0x40000 0x0 0x40000000 0x0 0x7e00000>;
-		interrupts = <GIC_PPI 0x0 IRQ_TYPE_EDGE IRQ_DEFAULT_PRIORITY>;
-		interrupt-parent = <&gic>;
-		status = "okay";
-	};
-
-	/*
-	 * This node may differs on different setups, because Xen picks
-	 * region for Domain-0 for every specific configuration. You can
-	 * start Xen for your platform and check following log:
-	 * (XEN) Allocating 1:1 mappings for dom0:
-	 * (XEN) BANK[0] 0x00000080000000-0x00000090000000 (256MB)
-	 *
-	 * Xen passes actual values for setup in domain device tree, but Zephyr
-	 * is not capable to parse and handle it in runtime.
-	 */
-	ram: memory@80000000 {
-		device_type = "mmio-sram";
-		reg = <0x00 0x80000000 0x00 DT_SIZE_M(256)>;
-	};
-};

